mungegithub/github/github.go

```diff
@@ -72,6 +72,15 @@ var (
 	combinedStatusLifetime = 5 * time.Second
 )
 
+func suggestOauthScopes(resp *github.Response, err error) error {
```

Comment:
Did we not want to add the check that OAuth scopes being missing was the root cause of the of 403?